### PR TITLE
Arabic translation

### PR DESCRIPTION
**Pull request by ihab-sensei**
Add Arabic translation and fix some translations in English

### CODE REVIEW

**Review by bertacsevercan — APPROVED** [review]

**Comment by ihab-sensei** [comment]
@halitbatur Can you merge Cihan's translations pr before this one.
There is going to be a conflict that i want to fix myself.

File: src/locales/ar/translation.json [hunk 1]
```diff
@@ -1,3 +1,247 @@
 {
-  "example": "مرحبا"
+  "example": "Hello",
```
Comment (halitbatur) [inline src/locales/ar/translation.json]: can we delete this line i dont think its used

File: src/locales/ar/translation.json [hunk 2]
```diff
@@ -1,3 +1,247 @@
...
+    }
+  },
+  "ourSponsorsComp": {
+    "card": {
+      "title0": "شركاؤنا"
+    }
+  },
+  "blog": {
+    "blog0": {
+      "title": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe.",
```
Comment (halitbatur) [inline src/locales/ar/translation.json:65]: these are no longer lorem so maybe we can translate whats written there

**Review by halitbatur — APPROVED** [review]

**Comment by halitbatur** [comment]
@ihab-sensei fix conflicts and its ready to be merged

File: src/locales/ar/translation.json (hunk 2 above)
Comment (ihab-sensei) [reply src/locales/ar/translation.json:65]: I actually changed back to lorem, since blog posts are shown as to how they were written by the user.

**Comment by ihab-sensei** [comment]
@halitbatur ptal!

**Pull request merged**